lib/services/consumptionManagement/lib/consumptionManagementClient.d.ts

```diff
@@ -40,14 +44,18 @@ declare class ConsumptionManagementClient extends AzureServiceClient {
    * @param {boolean} [options.generateClientRequestId] - When set to true a unique x-ms-client-request-id value is generated and included in each request. Default is true.
    *
    */
-  constructor(credentials: ServiceClientCredentials, subscriptionId: string, baseUri?: string, options?: AzureServiceClientOptions);
+  constructor(credentials: ServiceClientCredentials, subscriptionId: string, resourceGroupName: string, budgetName: string, baseUri?: string, options?: AzureServiceClientOptions);
```

Comment:
@shbha Why are resourceGroupName and budgetName properties of the client? Shouldn't they be method parameters? This will break existing clients.

Reply:
@shbha - The consumption api spec has been updated in this PR  https://github.com/Azure/azure-rest-api-specs/pull/2387. Please regenerate the sdk.